Libraries/LibKeyboard/CharacterMap.cpp

```diff
@@ -26,22 +26,65 @@
 
 #include "CharacterMap.h"
 #include <Kernel/Syscall.h>
-#include <LibKeyboard/CharacterMapFile.h>
+#ifndef KERNEL
+#    include <LibKeyboard/CharacterMapFile.h>
+#endif
 
 namespace Keyboard {
 
 CharacterMap::CharacterMap(const String& file_name)
 {
+#ifdef KERNEL
+    UNUSED_PARAM(file_name);
+    m_character_map_data = default_character_map;
+#else
     auto result = CharacterMapFile::load_from_file(file_name);
     ASSERT(result.has_value());
 
-    m_character_map = result.value();
+    m_character_map_data = result.value();
+#endif
 }
 
 int CharacterMap::set_system_map()
 {
-    Syscall::SC_setkeymap_params params { m_character_map.map, m_character_map.shift_map, m_character_map.alt_map, m_character_map.altgr_map };
+    Syscall::SC_setkeymap_params params { m_character_map_data.map, m_character_map_data.shift_map, m_character_map_data.alt_map, m_character_map_data.altgr_map };
     return syscall(SC_setkeymap, &params);
 }
 
+char CharacterMap::get_char(KeyEvent event)
+{
+    auto modifiers = event.modifiers();
+    auto index = event.scancode & 0xFF; // Remove E0 flah from scancode
```

Comment:
This comment is confusing, since `& 0xff` does not remove `0xe0` from something.
The bits being removed are `0xe000`. Also it's not really a flag, more like a scancode prefix.

Reply:
Index is last byte of scan code, e.g. for `0xE01D` index `0x1D`.